core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -496,7 +496,7 @@ public long getNumOfMessagesProcessed() {
 	@Override
 	public Receiver getReceiverByName(String receiverName) {
 		for (Receiver receiver: receivers) {
-			if (receiver.getName().equalsIgnoreCase(receiverName)) {
+			if (receiverName.equalsIgnoreCase(receiver.getName())) {
```

Comment:
receiverName can be null when getReceiverByName is called, e.g. from the GUI. 
We must avoid NullPointerExceptions!